src/include/daos/object.h

```diff
@@ -361,4 +362,13 @@ dc_obj_shard2anchor(daos_anchor_t *anchor, uint32_t shard)
 	anchor->da_shard = shard;
```

Comment:
this function is used by both client & server now, we probably should rename its prefix